Makefile

```diff
@@ -367,6 +367,12 @@ fix-abci-app-specs:
 	echo "Successfully validated abcis!"
 
 
-AEA_AGENT:=valory/hello_world:latest:$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
+AEA_AGENT_HELLO_WORLD:=valory/hello_world:latest:$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
+AEA_AGENT_ORACLE:=valory/oracle:latest:$(shell cat packages/hashes.csv | grep "agents/oracle" | cut -d "," -f2 )
+AEA_AGENT_APY_ESTIMATION:=valory/oracle:latest:$(shell cat packages/hashes.csv | grep "agents/apy_estimation," | cut -d "," -f2 )
```

Comment:
Why not `AEA_AGENT_APY_ESTIMATION:=valory/apy_estimation...`?

Reply:
Yup well spotted - that's wrong